gettsim/benefits/benefit_checks/benefit_checks.py

```diff
@@ -16,29 +16,24 @@ def arbeitsl_geld_2_m_minus_eink_hh(
 def wohngeld_vorrang_hh(
     wohngeld_vermögens_check_hh, arbeitsl_geld_2_m_minus_eink_hh,
 ):
-    rest_arbeitsl_geld_2 = (
-        arbeitsl_geld_2_m_minus_eink_hh - wohngeld_vermögens_check_hh
-    ).clip(lower=0)
-    return (rest_arbeitsl_geld_2 == 0) & (arbeitsl_geld_2_m_minus_eink_hh > 0)
+    out = wohngeld_vermögens_check_hh >= arbeitsl_geld_2_m_minus_eink_hh
```

Comment:
Delted check of claim.

Reply:
Why? ALG2 needs to be lower. `arbeitsl_geld_2_m_minus_eink_hh` might of course be zero.